garage/envs/box2d/car_parking_env.py

```diff
@@ -120,9 +119,9 @@ def action_from_keys(self, keys):
 
     @overrides
     def forward_dynamics(self, action):
-        if len(action) != flat_dim(self.action_space) + 1:
+        if len(action) != self.action_space.low.shape + 1:
```

Comment:
action_space.low.shape != flat_dim(action_space)

Reply:
I think needs `len()` because we're checking if the dimensions align but not the values themselves

Reply:
i.e. `if len(action.shape) != len(self.action_space.low.shape):`